### PR TITLE
CLOUDSTACK-8608. System VM's failed to start due to permissions issue.

### PR DESCRIPTION
**Pull request by likitha**
…instance.

### CODE REVIEW

**Comment by wilderrodrigues** [comment]
Travis failed due to a timeout.

I will have a look at the code.

File: plugins/hypervisors/vmware/src/com/cloud/hypervisor/vmware/manager/VmwareManagerImpl.java [hunk 1]
```diff
@@ -697,6 +699,22 @@ private String setupMountPoint(String parent) {
...
+            s_logger.debug("Set permissions for " + mountPoint);
+            String result = null;
+            Script script = new Script(true, "chmod", _timeout, s_logger);
+            script.add("-R", "0777", mountPoint);
+            result = script.execute();
+            if (result != null) {
+                s_logger.warn("Unable to set permissions for " + mountPoint + " due to " + result);
+            } else {
+                s_isSystemVmTmpltPermissionSet = true;
+            }
```
Comment (DaanHoogland) [inline plugins/hypervisors/vmware/src/com/cloud/hypervisor/vmware/manager/VmwareManagerImpl.java]: instance write-access to a static, even when in a synchronised block of code does not make for a good pattern. There is no guarantee that some other (synchronised) block accesses it as well.

**Comment by remibergsma** [comment]
Who wants to step in and finish this work? It seems the original author is not able to finish it. If no one steps in, we'll have to close the PR without merging it so please help :-).

**Comment by rhtyd** [comment]
tag:vmware-pickup

**Comment by sureshanaparti** [comment]
Continuing with the code changes here in a new PR #1875. This PR will be closed when PR #1875 is merged.

**Pull request closed without merging**